### PR TITLE
feat: clear xcode related junk script

### PR DESCRIPTION
**Pull request by japanese-goblinn**
## Description

- [x] New script command

## Screenshot

image

image

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/developer-utils/clear_xcode.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Optional parameters:
+# @raycast.icon 🧹
+#
+# Documentation:
+# @raycast.description This script clears all Xcode related junk.
+# @raycast.author Kirill Gorbachyonok
+# @raycast.authorURL https://github.com/japanese-goblinn
+
+rm -rf ~/Library/Caches/CocoaPods
+xcrun simctl delete unavailable
```
Comment (unnamedd) [inline commands/developer-utils/clear_xcode.sh:18]: Maybe you can also add a `rm -rf` to Carthage dependencies here as well, don’t you think?

Comment (japanese-goblinn) [reply commands/developer-utils/clear_xcode.sh:18]: > Maybe you can also add a `rm -rf` to Carthage dependencies here as well, don’t you think?

yep, nice idea, thanks

**Comment by fe9lix** [comment]
Hey guys, is this ready to be merged or anything still open...?

**Comment by unnamedd** [comment]
@fe9lix it's approved to me, as I did have only one request and @japanese-goblinn added it, we are ready to go!

**Pull request merged**